x-pack/plugins/reporting/public/management/__test__/report_listing.test.helpers.tsx

```diff
@@ -120,7 +121,12 @@ export type TestBed = Awaited<ReturnType<typeof setup>>;
 export const setup = async (props?: Partial<Props>) => {
   const uiSettingsClient = coreMock.createSetup().uiSettings;
   const httpService = httpServiceMock.createSetupContract();
-  const reportingAPIClient = new ReportingAPIClient(httpService, uiSettingsClient, 'x.x.x');
+  const reportingAPIClient = new ReportingAPIClient(
+    httpService,
+    uiSettingsClient,
+    'x.x.x',
+    mockConfig
```

Comment:
Per my comment for `reporting_api_client.ts`, it looks like we shouldn't need to add the new param for config, which should simplify the PR a bit.